tests/tests_geomstats/test_heisenberg.py

```diff
@@ -0,0 +1,55 @@
+"""Unit tests for the 3D heisenberg group in vector representation."""
+
+import geomstats.backend as gs
+import geomstats.tests
+from geomstats.geometry.heisenberg import HeisenbergVectors
+
+
+class TestHeisenbergVectors(geomstats.tests.TestCase):
+    def setUp(self):
```

Comment:
this should be setup_method, see: https://github.com/geomstats/geomstats/blob/6c3ed380c992d7f866448845e890ed198ce8bbea/tests/tests_geomstats/test_connection.py#L16